.github/workflows/pr-code-scan.yml

```diff
@@ -60,6 +60,9 @@ jobs:
           docker exec ${{ env.CONTAINER_NAME }} \
           bash -c "bash /GenAIExamples/.github/workflows/scripts/codeScan/${{ matrix.job_name }}.sh"
 
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@v2
```

Comment:
It's not a good idea to insert here. Here means it will be run inside `bandit` and `hadolint` test, this doesn't make sense.